fcm.go

```diff
@@ -205,56 +207,47 @@ func (this *FcmClient) sendOnce() (*FcmResponseStatus, error) {
 
 // Send to fcm
 func (this *FcmClient) Send() (*FcmResponseStatus, error) {
+
 	if this.Message.DryRun {
```

Comment:
All the code in this block is experimentation with firebase auth.
Cleaned up the logs a bit and have tried to make it such that it can be released to staging/prod